src/main/java/com/oltpbenchmark/benchmarks/templated/README.md

```diff
@@ -93,17 +93,17 @@ The distributions are dependent on the type of the value. Currently, the followi
 The following properties can be set on the value:
 
 -   `dist`: The distribution of the values
--   `min`: The minimum value the generator can produce. Defaults to 0
--   `max`: The maximum value the generator can produce. Defaults to 1
--   `seed`: A seed for the generator to ensure consistency
+-   `min`: The minimum value the generator can produce.
+-   `max`: The maximum value the generator can produce.
+-   `seed`: A seed for the generator to ensure consistency. Default is 0
 
-*NOTE:* For `Timestamp`, `Date`, and `Time` types, the min and max values must be converted to UNIX time (milliseconds).
+*NOTE:* For `Timestamp`, `Date`, and `Time` types, the min and max values must be converted to UNIX time (milliseconds) OR given as strings in the following format: (yyyy-MM-dd HH:mm:ss).
 The value is interpreted as the milliseconds since January 1, 1970, 00:00:00 GMT/UTC.
```

Comment:
```suggestion
The int value is interpreted as the milliseconds since January 1, 1970, 00:00:00 GMT/UTC.
The string value is also interpreted as UTC timezone.
```

Reply:
True?